src/renderer/components/NodeSelectorPanel/NodeSelectorPanel.tsx

```diff
@@ -246,24 +212,25 @@ export const NodeSelector = memo(() => {
                                                 );
                                             }
 
-                                            const hint = packageHints[category];
-                                            if (hint) {
+                                            if (category.installHint) {
                                                 return (
                                                     <RegularAccordionItem
                                                         category={category}
                                                         collapsed={collapsed}
-                                                        key={category}
+                                                        key={category.name}
                                                     >
                                                         <PackageHint
                                                             collapsed={collapsed}
-                                                            description={hint}
+                                                            hint={category.installHint}
+                                                            // TODO: Somehow link categories to deps
```

Comment:
TODO: Open issue instead of using a TODO.

---

Yeah, I think this should be in issue rather than a TODO comment.